pkgs/development/tools/build-managers/cmake/setup-hook.sh

```diff
@@ -93,6 +93,8 @@ fi
 
 addEnvHooks "$targetOffset" addCMakeParams
 
+export CLICOLOR_FORCE=1
```

Comment:
I think it makes sense to set this value here, because sometimes packages (e.g. the `cmake` one itself) use `dontUseCmakeConfigure`, but still use `cmake` with custom flags to configure it. These will have colored output too by exporting it here.